src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -339,6 +339,16 @@ injectRoute({
 });
 ```
 
+When publishing your package (`@fancy/dashboard`, in this case) to npm, you must export `dashboard.astro` in your `package.json`. Otherwise, the file will be looked up in the project folder, which will cause a `no such file or directory` exception.
```

Comment:
```suggestion
When publishing your package (`@fancy/dashboard`, in this case) to npm, you must export `dashboard.astro` in your `package.json`:
```
This is a case where I'm not sure all the negative information is helpful. If they follow the instructions, they shouldn't need to worry about how it could go wrong.